author: Andres Salomon <dilinger@debian.org>
description: switch back to the c++17 std

Upstream switched linux over to the c++20 for chromium 108, which breaks
linking for us because we're using gnu's libstdc++ instead of llvm's libc+++.

I'm not sure what to do about this long-term, but for now just switch back to
c++17.

--- a/build/config/compiler/BUILD.gn
+++ b/build/config/compiler/BUILD.gn
@@ -592,7 +592,7 @@ config("compiler") {
     } else if (is_linux) {
       # TODO(crbug.com/1284275): Switch to C++20 on all platforms.
       if (is_clang) {
-        cflags_cc += [ "-std=${standard_prefix}++20" ]
+        cflags_cc += [ "-std=${standard_prefix}++17" ]
       } else {
         # The gcc bots are currently using GCC 9, which is not new enough to
         # support "c++20"/"gnu++20".
